### PR TITLE
Bot can send stickers

### PR DESCRIPTION
**Pull request by andersann**
It send sticker when user sent some file(picture, document, sticker)

### CODE REVIEW

**Comment by andersann** [comment]
@allomov, you are my reviewer, can you comment this my task

**Comment by allomov** [comment]
@anatoliliotych could you take care of it?

File: answers.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module Answers
```
Comment (anatoliliotych) [inline answers.rb]: are you sure about using module?

Comment (andersann) [inline answers.rb]: Which way is better?

File: .env.example [hunk 2]
```diff
@@ -0,0 +1 @@
+TOKEN = 197540934:AAASVuLS7OvGbliEa4lyu96rri9SJVNn2bw
```
Comment (anatoliliotych) [inline .env.example]: I hope it's fake :)

File: answers.rb (hunk 1 above)
Comment (anatoliliotych) [inline answers.rb]: it's ambiguous to have module to store constant. move to main file. you can store this in file too and load.

File: .env.example (hunk 2 above)
Comment (andersann) [inline .env.example]: Yes, of course :)

File: answers.rb (hunk 1 above)
Comment (andersann) [inline answers.rb]: @anatoliliotych, first I stored these hashes in the main file, but then decided to move them to the modules, so as not to overload the file.

**Pull request merged**